presto-hive/src/main/java/com/facebook/presto/hive/HiveManifestUtils.java

```diff
@@ -149,6 +151,21 @@ public static long getFileSize(Page statisticsPage, int position)
         return partitionMetadata.build();
     }
 
+    public static long getManifestSizeInBytes(ConnectorSession session, PartitionUpdate partitionUpdate, Map<String, String> parameters)
+    {
+        if (isFileRenamingEnabled(session) && partitionUpdate.containsNumberedFileNames()) {
+            if (parameters.containsKey(FILE_NAMES) && parameters.containsKey(FILE_SIZES)) {
+                return parameters.get(FILE_NAMES).length() + parameters.get(FILE_SIZES).length();
+            }
```

Comment:
We just check if the extra params contains `MANIFEST_VERSION`. If yes, we sum up the values' lengths for `FILE_NAMES` and `FILE_SIZES`.